payloads/library/execution/PolyWog-Shell/payload.txt

```diff
@@ -0,0 +1,51 @@
+####################
+# Title: PolyWog-Shell
+# Author: Hak5Peaks
+# Catagory: Exacuation
+# Description: This payload uses a python2 + Bash script to generate a signature unique powershell rev shell that is typed out on the target PC.  
+# Target: Windows
+####################
+
+# PLEASE READ THE CONFIGUATION REQUIRMENTS INSIDE THE GITHUB REPO BEFORE RUNNING SCRIPT. 
```

Comment:
the bare minimum configuration requirements should ideally be within the comments of the payload -- payloads are meant to be portable and self containing remember